### ISSUE TITLE
get access to the return from within the console application?

### ISSUE THREAD

**Issue by andreireiand** [dotnet/docs#9780]
Take for example, this basic service exemplified here.
How would one grab hold of the addition result from within the:

    static void Main(string[] args)

function? 

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 0c8cbb16-05b1-a5f2-9031-1d7beb52efea
* Version Independent ID: 98092a9f-c19a-cca9-75d6-75b9c3d4cc0b
* Content: [How to: Host and Run a Basic Windows Communication Foundation Service](https://docs.microsoft.com/en-us/dotnet/framework/wcf/how-to-host-and-run-a-basic-wcf-service)
* Content Source: [docs/framework/wcf/how-to-host-and-run-a-basic-wcf-service.md](https://github.com/dotnet/docs/blob/master/docs/framework/wcf/how-to-host-and-run-a-basic-wcf-service.md)
* Product: **dotnet-framework**
* GitHub Login: @Lxiamail
* Microsoft Alias: **wcfsrvt**

**Comment by Thraka** [comment]
I don't fully understand what you are trying to do. Can you rephrase your question with some more detail?

Thanks
